clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -178,27 +171,11 @@ static void runAtClose(final Collection<Optional<? extends RequestManager>> requ
                            final NetworkClientDelegate networkClientDelegate,
                            final Timer timer) {
```

Comment:
Seems timer is now completely unused.

Reply:
noted, let's get rid of runAtClose all together in the future PR.